packages/webapp/src/components/Form/Checkbox/checkbox.module.scss

```diff
@@ -27,63 +60,39 @@
   width: 0;
 }
 
-/* Create a custom Checkbox */
-.checkmark {
-  position: relative;
-  height: 18px;
-  width: 18px;
-  background-color: var(--bgInputListTile);
-  border: solid 2px var(--grey600);
-  border-radius: 2px;
-}
-// Change the color of the Checkbox
-/* When the Checkbox is checked, add a green background */
-.container input:checked ~ .checkmark {
-  background-color: var(--teal700);
-  border: none;
-}
-
-/* Create the checkmark/indicator (hidden when not checked) */
-.checkmark:after {
-  content: '';
-  position: absolute;
+.checked {
   display: none;
 }
 
-/* Show the checkmark when checked */
-.container input:checked ~ .checkmark:after {
-  display: block;
-}
+.container input:checked {
+  ~ .checked {
+    display: block;
+  }
+  
+  ~ .unchecked {
+    display: none;
+  }
 
-/* Style the checkmark/indicator */
-.container .checkmark:after {
-  left: 5px;
-  width: 7px;
-  height: 13px;
-  border: solid white;
-  border-width: 0 2px 2px 0;
-  -webkit-transform: rotate(45deg);
-  -ms-transform: rotate(45deg);
-  transform: rotate(45deg);
+  ~ .label {
+    color: var(--Colors-Neutral-Neutral-900);
+    font-weight: 600;
```

Comment:
@kathyavini Not to always bold the label, I think we could pass a prop and apply a className conditionally like this.

```
...
  ~ .label {
    color: var(--Colors-Neutral-Neutral-900);
    // font-weight: 600;
  }
}

.container.shouldBoldSelected input:checked {
  ~ .label {
    font-weight: 600;
  }
}
```

Reply:
I love it, thank you!!! I was banging my head against the Select's styling forever instead of just fixing the new Checkbox 😄